macros/src/lib.rs

```diff
@@ -25,7 +25,7 @@ use proc_macro::TokenStream;
 use quote::quote;
 use syn::{self, parse_macro_input};
 
-// Custom derive macro for SizeOf trait.
+/// Custom-derive macro implementation for `SizeOf`.
```

Comment:
we can just say `Derive macro for [SizeOf]`. everything else seems superfluous. also, std adds a derivable section to the trait. maybe we should do the same. https://doc.rust-lang.org/std/clone/trait.Clone.html#derivable.

Reply:
will add the section.